docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 ARG STACTOOLS_VERSION=latest
 FROM ghcr.io/stac-utils/stactools:$STACTOOLS_VERSION
 
-WORKDIR /opt/stactools-package
+ARG DOCKER_PROJ_DIR_PARTIAL
```

Comment:
When I saw `PROJ` I thought we were building proj from scratch or something :-). It looks like this variable should be set to `stactools/threedep` or whatever, so maybe the variable could be `NAMESPACE_PACKAGE_DIRECTORY` or something similar to that? That's probably not the right name, curious if you have other ideas.

Reply:
Yeah, that one bothered me too. Everything I came up with was _very_ long.
Part of the issue is that passing `stactools/packagename` allows the rename to correctly use the underscore version of the name. Otherwise I'd just use the package name.

Reply:
It's not a huge deal as its pretty internal plumbing so shouldn't block, I can't think of a great alternative name either.

Reply:
The more I look at it, the more I think `PROJ` will come to haunt me.
`DOCKER_NAMESPACE_PACKAGE_DIR` is a bit unwieldy, but maybe it's better?